server/routes/eventDetail.ts

```diff
@@ -4,14 +4,12 @@ import { Router } from 'express';
 const eventDetailsRouter = Router();
 eventDetailsRouter.get('/', (req, res) => {
   const { id } = req.query;
-  console.log('in eventDetails');
   axios
     .get(
       `https://app.ticketmaster.com/discovery/v2/events.json?size=1&apikey=${process.env.TICKETMASTER_API_KEY}&id=${id}`
     )
     .then((results) => {
       const { data } = results;
```

Comment:
you can destructure this inside of the parameter on line 11 and get rid of line 12

Reply:
ok